### ISSUE TITLE
Unable to get error highlighting or auto-fix on .ts files

### ISSUE THREAD

**Issue by arnaudrenaud** [microsoft/vscode-eslint#684]
VS Code does not highlight any error on `.ts` files, and does not auto-fix them on save either.

VS Code does, however, highlight and auto-fix errors on `.js` files.
Furthermore, ESLint CLI does report errors on `.ts` files when running this command: `./node_modules/.bin/eslint src/ --ext .js,.ts`.

I use the following configuration:
`.eslintrc.js`:
```
module.exports = {
  parser: '@typescript-eslint/parser', // Specifies the ESLint parser
  extends: [
    'plugin:@typescript-eslint/recommended', // Uses the recommended rules from the @typescript-eslint/eslint-plugin
    'prettier/@typescript-eslint', // Uses eslint-config-prettier to disable ESLint rules from @typescript-eslint/eslint-plugin that would conflict with prettier
    'plugin:prettier/recommended', // Enables eslint-plugin-prettier and displays prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
  ],
  parserOptions: {
    ecmaVersion: 2018, // Allows for the parsing of modern ECMAScript features
    sourceType: 'module', // Allows for the use of imports
  },
};
```

`.settings.json`:
```
{
  "eslint.autoFixOnSave": true,
  "eslint.validate": [
    "javascript",
    "javascriptreact",
    { "language": "typescript", "autoFix": true },
    { "language": "typescriptreact", "autoFix": true }
  ],
  "editor.formatOnSave": true,
  "[javascript]": {
    "editor.formatOnSave": false
  },
  "[javascriptreact]": {
    "editor.formatOnSave": false
  },
  "[typescript]": {
    "editor.formatOnSave": false
  },
  "[typescriptreact]": {
    "editor.formatOnSave": false
  }
}
```

`package.json`:
```
    "@typescript-eslint/eslint-plugin": "^1.9.0",
    "@typescript-eslint/parser": "^1.9.0",
    "eslint": "^5.16.0",
    "eslint-config-prettier": "^4.3.0",
    "eslint-plugin-prettier": "^3.1.0",
    "prettier": "^1.18.0",
```

VS Code: 1.35.0
ESLint extension: 1.9.0

**Comment by dbaeumer** [comment]
Dups https://github.com/microsoft/vscode-eslint/issues/609

See https://github.com/microsoft/vscode-eslint/issues/609#issuecomment-460554105 for how to enable it.

There are some ideas how to improve this but the underlying problem is that the eslint plugin doesn't know that eslint got configure to validate TS Lint files.

**Comment by arnaudrenaud** [comment]
I already have the configuration specified in https://github.com/microsoft/vscode-eslint/issues/609#issuecomment-460554105 and it produces the outcome described above.

**Comment by dbaeumer** [comment]
Can you provide me with a GitHub repository that I can clone that demos this?

**Comment by arnaudrenaud** [comment]
I reproduce the error with this repo: https://github.com/arnaudrenaud/typescript-vscode-eslint-issue with VS Code 1.35.1 and VS Code ESLint 1.9.0.

**Comment by dbaeumer** [comment]
This works for me after moving the `settings.json` file into the `.vscode` folder which is its location. 

<img width="774" alt="capture" src="https://user-images.githubusercontent.com/1931590/62633197-0f70cf00-b934-11e9-9bcb-0cee3043e1b5.png">
